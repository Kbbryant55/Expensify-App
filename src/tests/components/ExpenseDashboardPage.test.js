import React from 'react';
import { shallow } from 'enzyme';
import expenses from '../fixtures/expenses';
import ExpenseDashBoardPage  from '../../components/ExpenseDashBoardPage';

test('Should render ExpenseDashBoardPage correctly', () => {
    const wrapper = shallow(<ExpenseDashBoardPage />);
    expect(wrapper).toMatchSnapshot();
});
