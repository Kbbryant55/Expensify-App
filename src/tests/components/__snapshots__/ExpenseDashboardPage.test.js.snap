// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render ExpenseDashBoardPage correctly 1`] = `
<div>
  <[object Object] />
  <[object Object] />
  <[object Object] />
</div>
`;
